src/deploy/NVA_build/Server.Dockerfile

```diff
@@ -146,31 +132,23 @@ RUN yum install -y -q bash \
 
 ##############################################################
 # Layers:
-#   Title: Node.js install with nvm
-#   Size: ~ 58 MB
-#   Cache: rebuild when Node.js version change in .nvmrc
-#
-# In order to build this we should run 
-# docker build from the local repo 
+#   Title: Getting the node 
+#   Size: ~ 110 MB
+#   Cache: Rebuild the .nvmrc is changing
 ##############################################################
-COPY ./.nvmrc ./noobaa-core/.nvmrc
-RUN export PATH=$PATH:/usr/local/bin && \
-    cd /usr/src && \
-    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.6/install.sh | bash && \
-    export NVM_DIR="/root/.nvm" && \
-    source /root/.nvm/nvm.sh && \
-    NODE_VER=$(cat /noobaa-core/.nvmrc) && \
-    nvm install ${NODE_VER} && \
-    nvm alias default $(nvm current) && \
-    cd ~ && \
-    ln -sf $(which node) /usr/local/bin/node && \
-    ln -sf $(which npm) /usr/local/bin/npm
+COPY ./.nvmrc ./.nvmrc
+RUN NODEJS_VERSION=v$(cat ./.nvmrc) && \
+    FILE_NAME=node-${NODEJS_VERSION}-linux-x64.tar.xz && \
+    cd / && \
+    curl https://nodejs.org/dist/${NODEJS_VERSION}/${FILE_NAME} > ${FILE_NAME} && \
+    tar -xf ${FILE_NAME} && \
+    ln -sf /node-${NODEJS_VERSION}-linux-x64/bin/node /usr/local/bin/node
```

Comment:
1. Doesn't make sense to me that this is extracted to `/`, use `/usr/local/node...` (refer to - https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard).
2. Use `ln -s` without force
3. Link also `npm` and `npx` from node bin.
4. Put this in a shell script - instead of repeating this code in multiple dockerfiles.